go/fn/object.go

```diff
@@ -981,6 +981,15 @@ func (o *SubObject) GetSlice(k string) SliceSubObjects {
 	return o.NestedSliceOrDie(k)
 }
 
+// SetSlice replaces the SliceSubObjects to the given field. It creates the field if not exists. If returns error if the field exists but not a slice type.
```

Comment:
```suggestion
// SetSlice sets the SliceSubObjects to the given field. It creates the field if it doesn't exist. IT returns error if the field exists but is not a slice type.
```

Description LGTM, just minor grammar nits